sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosBulkOperations.java

```diff
@@ -36,6 +36,24 @@ public static <T> CosmosItemOperation getCreateItemOperation(T item, PartitionKe
         return getCreateItemOperation(item, partitionKey, new CosmosBulkItemRequestOptions(), null);
     }
 
+    /**
+     * Instantiate an operation for Creating item in Bulk execution.
+     *
+     * @param id The id of the item for the operation.
+     * @param item A JSON serializable object that must contain an id property.
+     * @param partitionKey the partition key for the operation.
+     * @param <T> The type of item to be created.
+     *
+     * @return The bulk operation.
+     *
+     */
+    public static <T> CosmosItemOperation getCreateItemOperation(String id, T item, PartitionKey partitionKey) {
```

Comment:
Why are we exposing a new public API? Is this only for testing or do we want to expose this to customers?

Reply:
Removed it because realized it is not needed.